wiki/Glossary/en.md

```diff
@@ -334,41 +317,42 @@ Fail has three meanings:
 ### FC
 
 ![](img/CombosCanHurt.png)
-The image on the top has a full combo, while the one on the bottom has a broken combo.
+The score on the top has a full combo, while the one on the bottom has a broken combo.
 Losing your combo halfway through the game could easily lose you half as many points!
 
 FC or, a "full combo", is when a player completes a beatmap while obtaining the maximum combo possible.
 This does not account accuracy as it is often used to mean clearing a beatmap without having any misses or sliderbreaks.
 
-Full combos usually give the most score in the osu!standard and osu!catch modes. 
+Full combos usually give the most score in the osu!standard and osu!catch modes.
 
 ## G
 
 ### Geki
 
-Geki, or "Elite Beat!", is a scoring term used when you complete the comboset with 300s.
+Geki, or Elite Beat!, is a scoring term used when you complete the comboset with 300s.
```

Comment:
"is a scoring term used when you complete a combo with the highest accuracy on every note. Many geki will happen during the course of a successful play."